packages/grid/rollup.data-grid.config.js

```diff
@@ -40,6 +41,30 @@ export default [
     plugins: [
       dts(),
       !production && sourceMaps(),
+      production &&
+        copy({
+          targets: [
+            {
+              src: ['./data-grid/README.md', './data-grid/LICENSE'],
```

Comment:
**Off-topic**

The scripts in the core include the CHANGELOG too

Reply:
Added.